src/util/arith_tools.h

```diff
@@ -164,4 +164,9 @@ mp_integer power(const mp_integer &base, const mp_integer &exponent);
 void mp_min(mp_integer &a, const mp_integer &b);
 void mp_max(mp_integer &a, const mp_integer &b);
 
+/// get bit with given index from bit vector
+/// \param src: the bitvector representation
+/// \param bit_index: the index (0 is the least significant)
```

Comment:
I guess I don't usually complain about too much documentation, but I'm worried about inconsistencies if we document twice?

Reply:
removed